zh/course-03/lesson-01/chapter-12/README.md

```diff
@@ -0,0 +1,36 @@
+# 第十二章: 循环 
```

Comment:
Merge this with the previous chapter.

Chapter: Declare a winner or draw
https://beta.reactjs.org/learn/tutorial-tic-tac-toe#declaring-a-winner

Reply:
FIX